src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -586,6 +546,31 @@ def _configure_security_policy_storage_params(cmd):
 
 
 #####
+#           sql server aad-admin
+######
+
+
+with ParametersContext(command='sql server aad-admin create') as c:
+    c.expand('properties', ServerAzureADAdministrator)
+    c.argument('login',
+               options_list=('--user', '-u'))
+    c.argument('sid',
+               options_list=('--sid', '-s'))
+    c.argument('tenant_id',
+               options_list=('--tenant-id', '-t'))
+
+
+
+with ParametersContext(command='sql server aad-admin update') as c:
```

Comment:
Instead of repeating this, you can do this. This will make the params configration apply to anything under aad-admin

with ParametersContext(command='**sql server aad-admin**') as c:
    c.expand('properties', ServerAzureADAdministrator)
    c.argument('login',
               options_list=('--user', '-u'))
    c.argument('sid',
               options_list=('--sid', '-s'))
    c.argument('tenant_id',
               options_list=('--tenant-id', '-t'))